demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -385,36 +463,45 @@ def _collect_layouts_dependencies(pack_layouts, id_set):
             if packs_found_from_incident_indicator_fields:
                 pack_dependencies_data = PackDependencies. \
                     _label_as_mandatory(packs_found_from_incident_indicator_fields)
-                dependencies_packs.update(pack_dependencies_data)
+                layout_dependency.update(pack_dependencies_data)
+
+            verbose_file.write(
+                f'{os.path.basename(layout_data.get("file_path", ""))} depends on: '
+                f'{layout_dependency}  '
+            )
+            dependencies_packs.update(layout_dependency)
 
         return dependencies_packs
 
     @staticmethod
-    def _collect_incidents_fields_dependencies(pack_incidents_fields, id_set):
+    def _collect_incidents_fields_dependencies(pack_incidents_fields, id_set, verbose_file):
         """
         Collects in incidents fields dependencies.
 
         Args:
             pack_incidents_fields (list): collection of pack incidents fields data.
             id_set (dict): id set json.
+            verbose_file (VerboseFile): path to dependency explanations file.
 
         Returns:
             set: dependencies data that includes pack id and whether is mandatory or not.
 
         """
         dependencies_packs = set()
+        verbose_file.write('\n### Incident Fields')
 
         for incident_field in pack_incidents_fields:
             incident_field_data = next(iter(incident_field.values()))
+            incident_field_dependency = set()
 
-            related_incident_types = incident_field_data.get('incident_types', [])
-            packs_found_from_incident_types = PackDependencies._search_packs_by_items_names(
-                related_incident_types, id_set['IncidentTypes'])
-
-            if packs_found_from_incident_types:
-                pack_dependencies_data = PackDependencies. \
-                    _label_as_mandatory(packs_found_from_incident_types)
-                dependencies_packs.update(pack_dependencies_data)
+            # related_incident_types = incident_field_data.get('incident_types', [])
```

Comment:
Why did you comment this section?

Reply:
+1

Reply:
after speaking with @michalgold we agreed that this dependency is not correct.
fields should be based on types as they can exist without it.